BIDLicenses.js

```diff
@@ -38,33 +38,46 @@ const getCurrentLicense = async (licenseKey, serviceUrl, myKeyPair, requestUID =
         return infraKey;
     }
 
-    let cacheKey = `${serviceUrl}/${licenseKey}`;
-
-    let pubicKeyUrl = `${serviceUrl}/publickeys`;
-    let publicKey = (await WTM.executeRequest({
-        method: 'get',
-        url: pubicKeyUrl,
-        Logger,
-        requestUID,
-        cacheKey: pubicKeyUrl,
-        ttl: 600
-    })).json.publicKey;
-
-    let sharedKey = BIDECDSA.createSharedKey(myKeyPair.keySecret, publicKey);
-
     const requestId = JSON.stringify({
         ts: Math.round(new Date().getTime() / 1000),
         appid: senderId,
         uuid: requestUID
```

Comment:
@bill8575 for U1 route do you expect requestID to be JSON String OR just the UUID?